### PR TITLE
Update cli.md

### PR DESCRIPTION
**Pull request by 38elements**
@prateekbh

Would you please tell me for following question?

What does the following mean?

> Transparently code-split any component with an async! prefix.

`function asyncFoo {}` or `async function foo{}`

> Differential javascript serving.

Auto Loading or Hot Module Replacement

### CODE REVIEW

**Comment by developit** [comment]
Hi @38elements!

> Transparently code-split any component with an async! prefix.
>
> `function asyncFoo {}` or `async function foo{}`

Neither - this is referring to Preact CLI's import prefix. By prepending `"async!"` to an import specifier, it will be moved into a lazy-loaded bundle. The code is only downloaded and executed the first time `<Foo />` is rendered.

```js
import Foo from 'async!./foo';

export default function App() {
  return <div><Foo /></div>;
}
```

> Differential javascript serving.
>
> Auto Loading or Hot Module Replacement

This refers to Preact CLI's [module/nomodule](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/) output. It generates two sets of output bundles - one for modern browsers, and one for legacy browsers:

```html
<script type=module src="/main.12345.module.js"></script> <!-- Loaded by Chrome/Firefox/Safari/Edge -->
<script nomodule src="/main.12345.js"></script>  <!-- Loaded by IE 11, old Safari, etc -->
```

**Comment by 38elements** [comment]
@developit 
Thank you for answer.

File: content/en/cli.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Preact CLI comes with the following features out of the box.
 
 - 100/100 Lighthouse score, right out of the box ([proof](https://googlechrome.github.io/lighthouse/viewer/?gist=142af6838482417af741d966e7804346)).
 - Fully automatic code splitting for routes.
-- Transparently code-split any component with an async! prefix.
-- Differential javascript serving.
+- Using prefix "async!"  like `import Foo from 'async!./foo';`, Foo Component is lazy-loaded.
```
Comment (developit) [inline content/en/cli.md:21]:
```suggestion
- Lazy-load components by prefixing their import: `import Foo from 'async!./foo'`
```

File: content/en/cli.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ Preact CLI comes with the following features out of the box.
...
+- Generating two script tags for browser supporting to [module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and browser not supporting. ([More information](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/))
```
Comment (developit) [inline content/en/cli.md:22]:
```suggestion
- Outputs modern JavaScript with legacy fallback using the [module/nomodule pattern](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/)
```

**Comment by 38elements** [comment]
@developit 
Thank you for review.

**Pull request closed without merging**